### PR TITLE
FIN-3416 CollectionBuilder::addAll method added

### PR DESCRIPTION
**Pull request by vladimir-kot**
ISSUES RESOLVED
------
https://finsight.myjetbrains.com/youtrack/issue/FIN-3416

DESCRIPTION
--
According to this proposal https://github.com/dealroadshow/core-php/pull/597#discussion_r288925185, added new CollectionBuilder method for add whole Collection

QA
--
WhiteList and BlackList add/edit functionality for Roadshow should work as usual

### CODE REVIEW

File: lib/Collection/CollectionBuilder.php [hunk 1]
```diff
@@ -43,6 +43,19 @@ public function add($element): CollectionBuilder {
         return $this;
     }
 
+    /**
+     * @param Collection $collection
+     *
+     * @return CollectionBuilder
+     */
+    public function addAll(Collection $collection): CollectionBuilder {
```
Comment (nikolayfinsight) [inline lib/Collection/CollectionBuilder.php]: @Why addAll, and not addBulk as was before?

Comment (vladimir-kot) [reply lib/Collection/CollectionBuilder.php]: I researched util components and found that Java naming approach has been used in the library(see https://docs.oracle.com/javase/8/docs/api/java/util/Collection.html), including use of **addAll** method to add all elements from specified collection to current collection.
So in order to keep consistency I decided to leave Java naming approach in this case.

Ready to discuss it further if needed

Comment (nikolayfinsight) [reply lib/Collection/CollectionBuilder.php]: Good catch, thank you for your research!

**Comment by dogik** [comment]
@nikolayfinsight @vladimir-kot is there a plan on pointing core-php to dealroadshow/util, instead of granulephp/util repo? Does it make sense to incorporate util and data-bind into core-php repo?

**Comment by nikolayfinsight** [comment]
@dogik we discussed that with guys before but we don't have finalized decision for now. Possible variants which we discussed before:
Create a separate repo and move there all common parts which can reuse finsight and drs. Possible candidates: Query/Command dispatchers(for work with rabbitMQ bus); serialyzers(which convert VO into json and back when we transmit data between microservices over rabbitMQ bus); common utils like this and https://github.com/dealroadshow/data-bind
- include this separate repo as submodule or by the composer.json
As an example, we can discuss it with you when you will be in Kyiv, but for that moment, we decided to include only this repo into composer.json

**Comment by dogik** [comment]
@nikolayfinsight thanks!

**Pull request merged**